sections/header.liquid

```diff
@@ -270,7 +270,7 @@
     {%- if section.settings.logo_position == 'top-center' or section.settings.menu == blank -%}
       <details-modal class="header__search">
         <details>
-          <summary class="header__icon header__icon--search header__icon--summary link focus-inset modal__toggle" aria-haspopup="dialog" aria-label="{{ 'general.search.search' | t }}">
+          <summary class="header__icon header__icon--search header__icon--summary link focus-inset modal__toggle" aria-describedby="a11y-modal-window-message" aria-label="{{ 'general.search.search' | t }}">
```

Comment:
Similar to the product page, the search "modal" is implemented via the `summary`/`details`, which means the modal is still available when using `no-js` but without the focus trap and without `ESC` keyboard shortcut :)

Should we restrict this message to a JS scenario?